### PR TITLE
Adds logged_out_cta

### PR DESCRIPTION
**Pull request by dzucconi**
Closes https://github.com/aredotna/ervell/issues/986

### CODE REVIEW

**Comment by broskoski** [comment]
Hold, this needs a spruce up. Sorry, should have added a screenshot of what it looked like before.

**Comment by dzucconi** [comment]
This is what the confirm window looks like. So make it consistent.

**Comment by broskoski** [comment]
screen shot 2018-01-03 at 5 23 23 pm

File: components/logged_out_cta/index.coffee [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
     when 'profile'
-      "Join Are.na to follow #{USER.full_name}"
+      "Join Are.na to follow #{truncate(USER.full_name, 35)}"
     else
       'Are.na is a platform for creative and collaborative research'
 
   $('body').append $el = $ template call: call
+
+  $el.one 'click', '.js-close', (e) ->
+    e.preventDefault()
```
Comment (broskoski) [inline components/logged_out_cta/index.coffee]: Set cookie here so if you close it once, it never comes back.

**Pull request merged**